src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/test_scales_shifts_oob.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "common_test_utils/node_builders/constant.hpp"
+#include "utils/cpu_test_utils.hpp"
+
+using namespace ov::test;
+using namespace CPUTestUtils;
+
+namespace {
+
+// Checks that getScalesAndShifts no longer triggers out-of-bounds memory access 
+//tied to getPaddedElementsCount (regression for #32070).
+class ScalesShiftsOOBTest : public testing::WithParamInterface<std::string>,
+                            public SubgraphBaseStaticTest {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<std::string> obj) {
+        return "Device=" + obj.param;
+    }
+
+protected:
+    void SetUp() override {
+        targetDevice = this->GetParam();
+
+        ov::ParameterVector params;
+        auto input = std::make_shared<ov::op::v0::Parameter>(
+            ov::element::f32, 
+            ov::Shape{1, 1, 240, 320});
+        params.push_back(input);
+
+        // Conv layer
+        auto weights1 = ov::test::utils::make_constant(
+            ov::element::f32,
+            ov::Shape{24, 1, 3, 3});
+        auto conv1 = std::make_shared<ov::op::v1::Convolution>(
+            input,
+            weights1,
+            ov::Strides{1, 1},
+            ov::CoordinateDiff{1, 1},
+            ov::CoordinateDiff{1, 1},
+            ov::Strides{1, 1});
```

Comment:
Just FYI. We have convolution builders `src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/convolution.hpp`